### PR TITLE
XREAD command should accept 0 as a block argument

### LINKED ISSUE #135: XREAD command should accept 0 as a block argument
## Checklist

- Python 3.8.1
- Using hiredis
- Using asyncio event loop
- Issue exists in master

## Steps to reproduce

Calling this:
```
                streams = {stream: lastId}
                results = await connection.xread(count=None, block=0, **streams)
```

triggers this error:

`raise RedisError("XREAD block must be a positive integer")`

## Expected results

The command should not fail. The redis stream documentation uses 0 as a block parameter in its examples, in the stream intro section -> https://redis.io/topics/streams-intro

Screen Shot 2020-01-21 at 5 59 20 PM

### PR DESCRIPTION
**Pull request by bsergean**
## Description

Fix #135

### CODE REVIEW

**Comment by bsergean** [comment]
ping / I think that this one is a no brainer

File: aredis/commands/streams.py [hunk 1]
```diff
@@ -175,7 +175,7 @@ class StreamsCommandMixin:
         """
         pieces = []
         if block is not None:
-            if not isinstance(block, int) or block < 1:
+            if not isinstance(block, int) or block < 0:
                 raise RedisError("XREAD block must be a positive integer")
```
Comment (mateuszmandera) [inline aredis/commands/streams.py]: The error message should  change to "... non-negative integer" to be consistent.

**Comment by NoneGG** [comment]
Merged, thanks~

**Comment by mateuszmandera** [comment]
@NoneGG Please see my comment above, the ``RedisError`` message is inconsistent with the behavior, this can confuse people.

**Pull request merged**